compiler/noirc_evaluator/src/ssa/ir/dfg/simplify/binary.rs

```diff
@@ -345,40 +341,4 @@ mod tests {
         }
         ");
     }
-
-    #[test]
-    fn replaces_shl_on_zero_lhs_with_zero() {
```

Comment:
We could maybe keep these tests but asserting they don't change anything, explaining why we don't do this.

Reply:
I debated that but we could do that for any value so it felt strange to have this case up. But I could keep the test and add a comment.

Reply:
Makes sense 👍